src/Features/LanguageServer/Protocol/LanguageServerTarget.cs

```diff
@@ -131,6 +142,10 @@ public Task<InitializeResult> InitializeAsync(InitializeParams initializeParams,
 
                 Contract.ThrowIfTrue(_clientCapabilities != null, $"{nameof(InitializeAsync)} called multiple times");
                 _clientCapabilities = initializeParams.Capabilities;
+
+                // TO-DO: Add client capability check below once LSP side is merged
```

Comment:
Before merging this PR, will add a client capability check for refresh once LSP side is merged - https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient/pullrequest/388715